scripts/move_to_scala_2.10.sh

```diff
@@ -2,5 +2,17 @@
 
 set +x
 
-find . -name "pom.xml" -exec sed -e "s/2.11.8/2.10.6/g" -e "s/2.11/2.10/g" -i.2.10.bak '{}' \;
+grep "<scala\.version>" pom.xml | grep -q 2.10
+if [[ $? == 0 ]];
+then
+    echo "Scala version is already set to 2.10."
+    echo "Cowardly refusing to move to Scala 2.10..."
+
+    exit 1
+fi
+
+find . -name "pom.xml" -exec sed -e "s/2.11.8/2.10.6/g" \
+    -e "s/2.11/2.10/g" -i.2.10.bak \
+    -e "/no Scala/ s/Scala 2.10/Scala 2.11/g" -i.2.10.bak \
+    '{}' \;
 find . -name "*.2.10.bak" -exec rm {} \;
```

Comment:
generally, it might be good to put a -f flag on these files to ensure they get deleted.  completely optional, tho